apps/meeting/package.json

```diff
@@ -30,20 +29,14 @@
     "@types/react-router-dom": "^5.1.3",
     "@types/styled-system": "^5.1.9",
     "@types/throttle-debounce": "^2.1.0",
-    "concurrently": "5.2.0",
+    "@typescript-eslint/eslint-plugin": "^5.8.1",
+    "@typescript-eslint/parser": "^5.8.1",
     "css-loader": "^6.2.0",
-    "eslint": "^5.16.0",
-    "eslint-config-airbnb": "^17.1.1",
```

Comment:
Is there any reason we don't want to use this?

Reply:
I configured the eslint using command line where it asks for whether you want to use any preferred eslint configs like if to follow airbnb and so on. I checked the last deleted eslint config file and did not find any reference to airbnb rules hence removed it.

We were also not running lint so Instead of choosing the set solution, I used our own with basic set of rules.  With this setup I fixed around (12 errors and 34 warnings). I think we can make them more stricter as we want later. Still should I try airbnb config?

Reply:
no just want to check whether you think we should follow airbnb as it is quite popular. Anyway, we can always revisit this when we have a single centralize eslint config :)

Reply:
Let me do a quick check with airbnb setup and get back if I can fix all lets use it as a standard solution for now, if overwhelming errors and warning I will keep this setup for now 😅

Reply:
So, I got below errors with airbnb style guide:
✖ 447 problems (437 errors, 10 warnings)

We can re-visit this later 😅 .